cinema/serializers.py

```diff
@@ -1 +1,76 @@
-# write serializers here
+from rest_framework import serializers
+from rest_framework.relations import StringRelatedField
+
+from cinema.models import Actor, Genre, CinemaHall, Movie, MovieSession
+
+
+class ActorSerializer(serializers.ModelSerializer):
+    full_name = serializers.SerializerMethodField()
+
+    class Meta:
+        model = Actor
+        fields = ["id", "first_name", "last_name", "full_name"]
+
+    def get_full_name(self, actor):
+        return str(actor)
```

Comment:
I think here you could create property full_name in Actor model, then you don't need create function directly in serializer